Detections/AzureDiagnostics/MaliciousWAFSessions.yaml

```diff
@@ -0,0 +1,50 @@
+id: 46ac55ae-47b8-414a-8f94-89ccd1962178
+name: A potentially malicious web request was executed against a web server
+description: |
+  ‘Detects unobstructed Web Applicatio Firewall (WAF) activity in sessions where the WAF blocked incoming requests by computing the 
+  ratio between blocked requests and unobstructed WAF requests in these sessions (BlockvsSuccessRatio metric). A high ratio value for 
+  a given client IP and hostname calls for further investigation of the WAF data in that session, due to the significantly high number 
+  of blocked requests and a few unobstructed logs which may be malicious but have passed undetected through the WAF. The successCode 
+  variable defines what the detection thinks is a successful status code, and should be altered to fit the environment.’
+severity: Medium
+requiredDataConnectors:
+  - connectorId: WAF
+    dataTypes:
+      - AzureDiagnostics
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exploitation
```

Comment:
Exploitation is not a MITRE ATT&CK tactic.  Based on the Technique you selected, it should be InitialAccess - https://attack.mitre.org/techniques/T1190/